core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.jelly

```diff
@@ -24,12 +24,9 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <j:set var="previousIt" value="${it}" />
-  <j:new var="it" className="hudson.diagnosis.OldDataMonitor$ManagementLinkImpl" />
-
-  <l:settings-subpage>
-    <j:set var="it" value="${previousIt}" />
+  <j:new var="managementLink" className="hudson.diagnosis.OldDataMonitor$ManagementLinkImpl" />
 
+  <l:settings-subpage from="${managementLink}">
```

Comment:
What is the benefit of this over
```suggestion
  <l:settings-subpage from="hudson.diagnosis.OldDataMonitor$ManagementLinkImpl">
```
and having the `settings-subpage` handle instantiation (or preferably instance lookup)?

Are there use cases where we already have the instance ready?